lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -409,4 +475,47 @@ private ImageAssetBitmapManager getImageAssetBitmapManager() {
     }
     callback.unscheduleDrawable(this, what);
   }
+
+  private static class ColorFilterData {
+
+    final String layerName;
+    @Nullable final String contentName;
+    @Nullable final ColorFilter colorFilter;
+
+    ColorFilterData(String layerName, @Nullable String contentName,
+        @Nullable ColorFilter colorFilter) {
+      this.layerName = layerName;
+      this.contentName = contentName;
+      this.colorFilter = colorFilter;
+    }
+
+    @Override public int hashCode() {
```

Comment:
Since these objects are being put into a set, I believe I do need these. Allows LottieDrawble to not perform unnecessary work.